stem/model.py

```diff
@@ -1,6 +1,7 @@
-from typing import List, Sequence, Dict, Any, Optional, Union
 from enum import Enum
 from dataclasses import dataclass
+import collections
```

Comment:
I think collections import is not needed, since it is already done in the typing import